R/data.table.R

```diff
@@ -12,6 +12,10 @@ methods::setPackageName("data.table",.global)
 #   (2) export() in NAMESPACE
 #   (3) add to vignettes/datatable-importing.Rmd#globals section
 .SD = .N = .I = .GRP = .NGRP = .BY = .EACHI = NULL
+. = function(...) NULL
+J = function(...) NULL
+patterns = function(...) NULL
+measure = function(...) NULL
 # These are exported to prevent NOTEs from R CMD check, and checkUsage via compiler.
 # But also exporting them makes it clear (to users and other packages) that data.table uses these as symbols.
 # And NULL makes it clear (to the R's mask check on loading) that they're variables not functions.
```

Comment:
"NULL makes it clear ... that they're variables not functions" should probably be updated now, since some of these are functions.

Reply:
should i just remove this line or we can replace it with something like "# The presence of `NULL` signals to R's mask check during loading that these are entities with specific behavior. However, it's worth noting that the inclusion of some new masks now encompasses functions as well."

Reply:
I have updated the comment, if there is need of further changes for this PR plz suggest :)